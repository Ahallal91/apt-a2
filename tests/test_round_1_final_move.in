BRLURBBBBRYUYBLLBBRYRBLYLUURBUUYBYRRRUBUYUBBBRUUYBRULRYUBURRYUYYRUBLYUUYLLRLYRRLYYYLLYLLLLLBULBURLRY
Test Player 1
Test Player 2
turn 5 B 2
turn 2 B 3
turn 4 L 3
turn 0 R 2
turn 0 Y 4
turn 1 R 4
turn 0 L 1
turn 3 R 4
turn 0 B 5
turn 0 U 1
