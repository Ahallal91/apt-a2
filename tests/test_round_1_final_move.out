=== Round 1 ===

Factories:
0: Y 
1: 
2: 
3: 
4: 
5: 

Score for Test Player 1: 0
Mosaic for Test Player 1:
1:         L || . . . . .
2:       B B || . . . . .
3:     . L L || . . . . .
4:   . . Y Y || . . . . .
5: . . B B B || . . . . .
6: broken |

Score for Test Player 2: 0
Mosaic for Test Player 2:
1:         U || . . . . .
2:       R R || . . . . .
3:     B B B || . . . . .
4:   . . R R || . . . . .
5: . . . . . || . . . . .
6: broken | F U

